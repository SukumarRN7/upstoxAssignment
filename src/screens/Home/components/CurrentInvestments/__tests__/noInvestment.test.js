import React from 'react';
import renderer from 'react-test-renderer';
import NoInvestments from '../noInvestments';

describe('NoInvestments', () => {
  test('renders correctly', () => {
    const component = renderer.create(<NoInvestments />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
