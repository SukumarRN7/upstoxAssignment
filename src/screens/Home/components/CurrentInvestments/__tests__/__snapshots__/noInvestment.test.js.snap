// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoInvestments renders correctly 1`] = `
<View
  style={
    [
      {
        "flex": 1,
      },
      {
        "alignItems": "center",
        "justifyContent": "center",
      },
    ]
  }
>
  <Text>
    You have no investments
  </Text>
</View>
`;
